packages/frontend/public/api-doc.yml

```diff
@@ -32,6 +44,11 @@ paths:
             enum:
               - timestamp
               - upvotes
+        - name: includeReplies
```

Comment:
@lsankar4033  Sorry for the late comment but I believe we want to set the name to `rootOnly` here?

Reply:
Oh shoot you're right